README.md

```diff
@@ -25,7 +26,8 @@ Enter your bot token and all correct informations:
             "guildID": "111111",
             "channelID": "121212",
             "whID": "232323",
-            "whToken": "webhookToken2323"
+            "whToken": "webhookToken2323",
+            "ignore": false
```

Comment:
typo here as you named it ignore and not ignored

Reply:
Not a typo. I made it `ignore` to have it make more sense to the end user.

Reply:
faire enough.Indeed my bad. However it's called ignored in the template.